pkg/message/raw.go

```diff
@@ -0,0 +1,32 @@
+package message
+
+import (
+	"fmt"
+
+	"github.com/golang/glog"
+	"github.com/sbezverk/gobmp/pkg/bmp"
+)
+
+// produceStatsMessage proceduces message from BMP Statistic Message
+func (p *producer) produceRawMessage(msg bmp.Message) {
+	if msg.PeerHeader == nil {
+		// XXX
+		glog.Errorf("perPeerHeader is missing, cannot construct proper raw message")
+		return
+	}
+	rm, ok := msg.Payload.(*bmp.RawMessage)
+	if !ok {
+		glog.Errorf("got invalid Payload type in bmp.RawMessage %+v", msg.Payload)
+		return
+	}
+
+	out := []byte(fmt.Sprintf("V: 1.7\nC_HASH_ID: %s\nR_HASH: %s\nR_IP: %s\nL: %d\n\n",
```

Comment:
Sprintf in golang is very costly,  please try to optimize the process of building the message.